src/app/movieList/movieCard/movieCard.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
```

Comment:
not  sure we need for rows for each card, bat leave it as it is now. will refactor later